### PR TITLE
Collection camp UI update

### PR DESCRIPTION
**Pull request by belwalshubham**
Collection camp UI updates

### CODE REVIEW

File: wp-content/themes/goonj-crm/functions.php [hunk 1]
```diff
@@ -95,8 +95,36 @@ function goonj_custom_password_reset_redirection( $errors, $user ) {
 	}
 }
 
-add_shortcode( 'goonj_check_user_form', 'goonj_check_user_action' );
+add_shortcode('goonj_check_user_form', 'goonj_check_user_action');
```
Comment (Satendra-SR) [inline wp-content/themes/goonj-crm/functions.php:98]: Can you do it in this way only?

```suggestion
add_shortcode( 'goonj_check_user_form', 'goonj_check_user_action' );
```

File: wp-content/themes/goonj-crm/functions.php [hunk 2]
```diff
@@ -241,6 +270,13 @@ function goonj_custom_message_placeholder() {
 }
 add_shortcode('goonj_volunteer_message', 'goonj_custom_message_placeholder');
 
+function goonj_collection_camp_landing_page() {
+    ob_start();
+    get_template_part('templates/collection-landing-page');
+    return ob_get_clean();
+}
+add_shortcode('goonj_collection_landing_page', 'goonj_collection_camp_landing_page');
```
Comment (Satendra-SR) [inline wp-content/themes/goonj-crm/functions.php:278]:
```suggestion
add_shortcode( 'goonj_collection_landing_page', 'goonj_collection_camp_landing_page' );
```

File: wp-content/themes/goonj-crm/assets/styles/main.scss [hunk 3]
```diff
@@ -293,6 +294,31 @@ form input {
...
+    .mobile-margins-l24-r30 {
+        margin-left: 24px !important;
+        margin-right: 30px !important;
+        margin-bottom: 30px !important;
+    }
+    .mobile-margin-left-30 {
+        margin-left: 30px !important;
+    }
+
+    .mobile-margins-l30-r30 {
```
Comment (Satendra-SR) [inline wp-content/themes/goonj-crm/assets/styles/main.scss:316]: what does this mean? 

From what I can read, it is for mobile screens, but you are putting it inside the 768px breakpoints.

Also, we don't have to specify margins class for each size indipendently, I believe we have $spacers and $sizes defined in `variable.scss` which can be used directly. 

Checkout this, which can help you to get more details on how these things works - https://getbootstrap.com/docs/4.0/utilities/spacing/

**Review by Satendra-SR — CHANGES_REQUESTED** [review]
@belwalshubham added a few comments, can you check once?

File: wp-content/themes/goonj-crm/assets/styles/main.scss (hunk 3 above)
Comment (Satendra-SR) [reply wp-content/themes/goonj-crm/assets/styles/main.scss:316]: @belwalshubham I read it `min-width`.

Actually you don't have to specify`max-width`, instead, override it in md.scss file

Comment (belwalshubham) [reply wp-content/themes/goonj-crm/assets/styles/main.scss]: fixed

File: wp-content/themes/goonj-crm/assets/styles/main.scss [hunk 4]
```diff
@@ -293,6 +294,31 @@ form input {
...
+        margin-left: 30px !important;
+        margin-right: 30px !important;
+    }
+}
```
[the remark this answers is not in the document]
Comment (belwalshubham) [reply wp-content/themes/goonj-crm/assets/styles/main.scss]: fixed

**Review by Satendra-SR — APPROVED** [review]

**Pull request merged**